docs/_posts/2015-01-27-react-v0.13.0-beta-1.md

```diff
@@ -135,17 +135,19 @@ You can also use CoffeeScript classes:
 div = React.createFactory 'div'
 
 class Counter extends React.Component
-  @propTypes =
-    initialCount: React.PropTypes.number
-  @defaultProps =
-    initialCount: 0
-  constructor: ->
-    @state =
-      count: @props.initialCount
+  @propTypes = initialCount: React.PropTypes.number
+  @defaultProps = initialCount: 0
+
+  constructor: (props) ->
+    super props
+    @state = count: props.initialCount
+
   tick: =>
     @setState count: @state.count + 1
+
   render: ->
-    div(onClick: @tick, 'Clicks: ', @state.count)
+    div onClick: @tick,
+      "Clicks: #{@state.count}"
```

Comment:
This is actually different than what we have in the ES6 class above. Doing the string interpolation like this results in a single text child, where as what we do will result in 2.

The rest seems fine.